FixedPopupPanels/tests/FixedPopupPanels-specs.js

```diff
@@ -1,43 +1,61 @@
-import {mount} from 'enzyme';
+import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import {render} from '@testing-library/react';
 
 import FixedPopupPanels from '../FixedPopupPanels';
 
+const FloatingLayerController = FloatingLayerDecorator('div');
+
 describe('FixedPopupPanels', () => {
 
 	it('should have the default width when nothing is assigned', function () {
-		const subject = mount(<FixedPopupPanels open />);
+		const {getByTestId} = render(
+			<FloatingLayerController>
+				<FixedPopupPanels data-testid="panels-id" open />
+			</FloatingLayerController>
+		);
 
 		const expected = 'narrow';
-		const actual = subject.find('Popup').prop('className');
+		const actual = getByTestId('panels-id').parentElement.parentElement;
 
-		expect(actual).toContain(expected);
+		expect(actual.className).toContain(expected);
```

Comment:
Please add .classname to the const declaration not in the expect
(same for the ones below)